crates/oxc_allocator/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    println!("cargo:rustc-check-cfg=cfg(nightly)");
```

Comment:
This is required to prevent a warning on `#[cfg(nightly)]`. This effectively adds `nightly` to the list of allowed cfg-attributes for the crate.